diff --git a/index.js b/index.js
--- a/index.js
+++ b/index.js
@@ -1,4 +1,4 @@
-const Minipass = require('minipass')
+const { Minipass } = require('minipass')
 const _flush = Symbol('_flush')
 const _flushed = Symbol('_flushed')
 const _flushing = Symbol('_flushing')
diff --git a/package.json b/package.json
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "tap": "^15.1.6"
   },
   "dependencies": {
-    "minipass": "^3.0.0"
+    "minipass": "^7.0.0"
   },
   "files": [
     "index.js"
